Makefile

```diff
@@ -198,7 +198,7 @@ generate: controller-gen
 # end-to-tests
 .PHONY: e2e
 e2e: chainsaw
-	$(CHAINSAW) test --test-dir ./tests/e2e
+	$(CHAINSAW) test --test-dir ./tests/e2e/ingress
```

Comment:
I think this might've been committed by mistake.

Reply:
Right, but already removed.